blogposts/2024-05-15-improved-gallery-fabric-0.74.md

```diff
@@ -0,0 +1,102 @@
+
+## New Features
+
+### View Managers Empowered to Capture Pointers 🖱️ https://github.com/microsoft/react-native-windows/pull/8969
+
+Now, view managers can control of the pointer with the new 'AllowUncaptured' flag. This feature enables continued receipt of events, such as PointerMoved, even after the initial press. Say hello to more sophisticated interactions, like dragging elements beyond their conventional boundaries. [Learn more...](https://github.com/microsoft/react-native-windows/pull/8969)
+
+### Streamlined C++ Template Support for create-react-native-library 
+
+Introducing a fresh template, seamlessly integrating with the 'npx create-react-native-library' command. Say goodbye to complexities; now adding Windows support for turbo modules is a breeze. [Learn more...](https://github.com/microsoft/react-native-windows/pull/12481)
+
+### React Native Windows Upgraded to C++20  
+
+With React Native Windows now adopting the latest C++20 standard, we're bringing our codebase up to speed with modern development practices. [Learn more...](https://github.com/microsoft/react-native-windows/pull/12656)
+
+### Enable custom timer factory implementation 
+
+Introducing `Microsoft.ReactNative.ReactCoreInjection.SetTimerFactory`, a game-changer for custom timer implementations in React Native for Windows. Ideal for environments lacking a `Microsoft.UI.Dispatching.DispatcherQueue`, this feature ensures seamless operation, even under diverse circumstances. [Learn more...](https://github.com/microsoft/react-native-windows/pull/12856)
+
+
+### Enhanced Pressable Component with Disabled Property Support
+
+In response to community feedback, we've bolstered the Pressable component's functionality. This update introduces native support for the `disabled` property, while also ensuring that the onKeyDown event handler behaves appropriately when the component is disabled. [Learn more...](https://github.com/microsoft/react-native-windows/pull/12799)
+
+## Deprecations
+
+### Direct Debugging as the Default
+
+To streamline our debugging process, we're transitioning away from web debugging to align with core practices. For all engines except Hermes, we recommend utilizing direct debugging. While Hermes currently experiences challenges with direct debugging, rest assured, we're actively working on a solution for an upcoming patch release. Check for updates surrounding Hermes [here](https://github.com/microsoft/react-native-windows/issues/12982)
+
+### CoreApp APIs and prototypes
+
+CoreApp was created as a testing ground to explore potential directions for Fabric. However, as we pivoted towards a different trajectory, we've made the decision to deprecate these APIs.
+
+## How can I upgrade my App to get all this good stuff??
```

Comment:
maybe a notch too informal? Doesn't seem to match the tone of the rest of the blogpost